src/main/java/com/sonyericsson/jenkins/plugins/bfa/db/MongoDBKnowledgeBase.java

```diff
@@ -482,6 +484,16 @@ public void saveStatistics(Statistics stat) throws UnknownHostException, Authent
             cause.put("build", upstreamCause.getUpstreamBuild());
         }
         object.put("upstreamCause", cause);
+
+        DBObject parameters = null;
+        if (stat.getParameters() != null) {
+            parameters = new BasicDBObject();
+            for(ParameterValue item: stat.getParameters()) {
+              parameters.put(item.getName(), item.getValue());
```

Comment:
`item.getValue()` can return a complex object graph, is that really what you want? Wouldn't for example the [realized env vars](http://javadoc.jenkins.io/hudson/model/ParameterValue.html#buildEnvironment-hudson.model.Run-hudson.EnvVars-) from those parameters be better to have?